samples/queries.js

```diff
@@ -20,41 +20,42 @@ function printResult(rows) {
   // [START bigquery_simple_app_print]
   console.log('Query Results:');
   rows.forEach(function(row) {
-    let str = '';
-    for (let key in row) {
-      if (str) {
-        str = `${str}\n`;
-      }
-      str = `${str}${key}: ${row[key]}`;
-    }
-    console.log(str);
+    let url = row['url'];
+    let viewCount = row['view_count'];
+    console.log(`url: ${url}, ${viewCount} views`);
   });
   // [END bigquery_simple_app_print]
 }
 
-function queryShakespeare(projectId) {
-  // [START bigquery_simple_app_query]
+function queryStackOverflow(projectId) {
+  // [START bigquery_simple_app_deps]
   // Imports the Google Cloud client library
   const BigQuery = require('@google-cloud/bigquery');
+  // [END bigquery_simple_app_deps]
 
+  // [START bigquery_simple_app_client]
   /**
    * TODO(developer): Uncomment the following lines before running the sample.
    */
   // const projectId = "your-project-id";
 
-  // The SQL query to run
-  const sqlQuery = `SELECT
-    corpus, COUNT(*) as unique_words
-    FROM publicdata.samples.shakespeare
-    GROUP BY
-      corpus
-    ORDER BY
-    unique_words DESC LIMIT 10;`;
-
   // Creates a client
   const bigquery = new BigQuery({
     projectId: projectId,
   });
+  // [END bigquery_simple_app_client]
+
+  // [START bigquery_simple_app_query]
+  // The SQL query to run
+  const sqlQuery = `SELECT
+    CONCAT(
+      'https://stackoverflow.com/questions/',
+      CAST(id as STRING)) as url,
+    view_count
+    FROM \`bigquery-public-data.stackoverflow.posts_questions\`
```

Comment:
The auto-linter (`npm run prettier`) is getting a bit tripped up over this line. Escaping seems the most natural solution, but feel free to suggest an alternative.

Reply:
Escaping works for me.